src/commands/utility/docs.js

```diff
@@ -0,0 +1,24 @@
+const { Command } = require('../../')
+const fetch = require('node-fetch')
+
+module.exports = class Docs extends Command {
+  constructor (client) {
+    super({
+      name: 'docs',
+      category: 'utility',
+      parameters: [{
+        type: 'string',
+        full: true,
+        missingError: 'commands:docs.noQueryProvided'
+      }]
+    }, client)
+  }
+
+  async run ({ channel }, query) {
+    const queryParams = new URLSearchParams({ src: 'stable', q: query })
```

Comment:
What if i dont want search in the stable branch?

Reply:
I'm going to hop in and suggest using a flag like `--branch` to choose a branch.